app/views/course/assessment/submission/submissions/_buttons.html.slim

```diff
@@ -0,0 +1,23 @@
+- submission = f.object
+div.submission-buttons
+  - if defined?(save) && save
```

Comment:
Seems it's a private API and was removed: https://github.com/rails/rails/commit/3dfcae6afa24b641bd838b9044c5ce9aa2a1a6db

So rails 5 adds it back ?